sinabs/backend/dynapcnn/dynapcnn_network.py

```diff
@@ -320,172 +523,121 @@ def make_config(
-
-        Args:
-            device_type (str): Device type ie speck2b, speck2fmodule
+        config_builder = ChipFactory(device).get_config_builder()
 
-        Returns:
-            bool: True if compatible
-        """
-        try:
-            _, is_compatible = self._make_config(device=device_type)
-        except ValueError as e:
-            # Catch "No valid mapping found" error
-            if e.args[0] == ("No valid mapping found"):
-                return False
+        if chip_layers_ordering is not None:
+            if layer2core_map is not None:
+                warn(
+                    "Both `chip_layers_ordering` and `layer2core_map are provided. "
+                    "Please only provide `layer2core_map`, as `chip_layers_ordering` "
+                    "is deprecated.",
+                    DeprecationWarning,
+                )
+            elif chip_layers_ordering == "auto":
+                warn(
+                    "The parameter `chip_layers_ordering` is deprecated. Passing "
+                    "'auto' is still accepted, but in the future please use "
+                    "`layer2core_map` instead.",
+                    DeprecationWarning,
+                )
+                layer2core_map = "auto"
             else:
-                raise e
-        return is_compatible
-
-    def reset_states(self, randomize=False):
-        """Reset the states of the network."""
-        if hasattr(self, "device") and isinstance(self.device, str):  # pragma: no cover
-            device_name, _ = parse_device_id(self.device)
-            if device_name in ChipFactory.supported_devices:
-                config_builder = ChipFactory(self.device).get_config_builder()
-                # Set all the vmem states in the samna config to zero
-                config_builder.reset_states(self.samna_config, randomize=randomize)
-                self.samna_device.get_model().apply_configuration(self.samna_config)
-                # wait for the config to be written
-                time.sleep(1)
-                # Note: The below shouldn't be necessary ideally
-                # Erase all vmem memory
-                if not randomize:
-                    if hasattr(self, "samna_input_graph"):
-                        self.samna_input_graph.stop()
-                        for lyr_idx in self.chip_layers_ordering:
-                            config_builder.set_all_v_mem_to_zeros(
-                                self.samna_device, lyr_idx
-                            )
-                            time.sleep(0.1)
-                        self.samna_input_graph.start()
-                return
-        for layer in self.sequence:
-            if isinstance(layer, DynapcnnLayer):
-                layer.spk_layer.reset_states(randomize=randomize)
-
-    def find_chip_layer(self, layer_idx):
-        """Given an index of a layer in the model, find the corresponding cnn core id where it is
-        placed.
+                raise ValueError(
```

Comment:
No need for exception here. Can convert chip layers ordering to layer2core map and just raise warning.